x/staking/keeper/keeper.go

```diff
@@ -93,3 +97,49 @@ func (k Keeper) SetLastTotalPower(ctx sdk.Context, power sdk.Int) {
 	bz := k.cdc.MustMarshal(&sdk.IntProto{Int: power})
 	store.Set(types.LastTotalPowerKey, bz)
 }
+
+// GetValidatorUpdate returns the ABCI validator power update for the current block
+// by the consensus address.
```

Comment:
So this is the ABCI power update for a certain block for a certain validator?

Reply:
Yes though CCV just needs `GetValidatorUpdates`